api/main.py

```
@@ -36,6 +36,18 @@
     # Get port from environment variable or use default
     port = int(os.environ.get("PORT", 8001))
 
+    # Explicitly set environment variables for worker processes
+    import multiprocessing
```

Ruff (0.11.9)

`multiprocessing` imported but unused

Remove unused import: `multiprocessing`

(F401)